### DISCUSSION TITLE
Output Mapping "problem"

### DISCUSSION THREAD

**Discussion by dcan999** [iNavFlight/inav#9749] [General]
The S1 and S2 output pads ripped off my FC, so I thought, I will use the new output mapping feature in INAV 7 to fix it! 

I originally had S1 as motor, and S3 and S4 as servos (flying wing). 

I thought I would just change Timer 2 to MOTORS and use S5 for motor output, and keep using S3 and S4 (Timer 1) as servos. 

First I had to change Timer 8 (S1 and S2) to SERVOS so it wouldn't automatically assign the motor to them. 

Then, I changed Timer 2 to MOTORS. 

This successfully changed my motor output to S5

The problem is, it is auto-assigning my Servos 1 and 2 to Timer 8 (S1/S2) which are the pads I can't use. 

If I make Timer 8 MOTORS again, it automatically assigns the motor output to S1 instead of S5 again, so that doesn't work. 

I tried making more mixer rules and changing the servo numbers around but no matter what I do in regards to SERVOS/MOTORS settings on all the timers, it keeps assigning servos 1 and 2 to S1/S2. 

It seems like it just goes in order to the first available Timer with the proper settings when assigning outputs, which happens to be my broken S1/S2....

Is there a workaround for this or something I am missing?

**Comment by MrD-RC** [comment]
This is normal and working as expected. You will need to create two servo outputs and set them as indexes 1 and 2. Then continue the actual servos from index 3

**Reply by dcan999** [reply]
Thanks, I forgot to ask if the servos HAD to match what the diagram was showing for servos, since it shows S1 and S2 as the elevons and it won't change, that's what made me think it wasn't working, but I guess the S1 and S2 will just be redundant and as long as I have 3 and 4 as pitch and roll it will still work, the diagram just won't be correct.

**Reply by MrD-RC** [reply]
If you set 1 & 2 to be something other than stabilised pitch/roll/etc. The diagram should update.  Try setting them to use the max input type.

**Reply by dcan999** [reply]
That worked and made the diagram match, I knew it had to be some macgyvering needed. 

Subbed to your patreon as thanks!
